server/verify.go

```diff
@@ -77,14 +78,22 @@ func VerifyAttestation(attestation *pb.Attestation, opts VerifyOpts) (*pb.Machin
 			continue
 		}
 
-		// Parse the event log and replay the events against the provided PCRs
+		// Parse event logs and replay the events against the provided PCRs
 		pcrs := quote.GetPcrs()
-		state, err := ParseMachineState(attestation.GetEventLog(), pcrs)
+		state, err := parsePCClientEventLog(attestation.GetEventLog(), pcrs)
 		if err != nil {
-			lastErr = fmt.Errorf("failed to validate the event log: %w", err)
+			lastErr = fmt.Errorf("failed to validate the PCClient event log: %w", err)
 			continue
 		}
 
+		celState, err := parseCanonicalEventLog(attestation.CanonicalEventLog, pcrs)
```

Comment:
Nit: `attestation.GetCanonicalEventLog()` for consistency with above code

Reply:
Done